components/p3a/buildflags.gni

```diff
@@ -1,3 +1,20 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
 declare_args() {
-  brave_p3a_enabled = true
+  brave_p3a_enabled = false
```

Comment:
Why is this defaulting to false now? How can p3a tests run locally if p3a is disabled by default? Also this will disable it by default even for official builds which is definitely not right